.github/workflows/tests_qiskit_1.yml

```diff
@@ -0,0 +1,50 @@
+name: Tests for 1.0
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.9, '3.10', '3.11']
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.4.1
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements-ci0.txt
+          pip install wheel pytest pytest-cov pytest-mock flaky --upgrade
+
+      - name: Install Plugin
+        run: |
+          python setup.py bdist_wheel
+          pip install dist/PennyLane*.whl
+
+      - name: Run Qiskit converter tests
+        # Test conversion to PennyLane with Qiskit 1.0.0
+        run: python -m pytest tests/test_converter.py
+        env:
+          IBMQX_TOKEN: ${{ secrets.IBMQX_TOKEN_TEST }}

```

Comment:
Out of curiosity, do we need this environment variable?

Reply:
needed for tests that use the circuit-runner or sampler devices (or any bespoke test device that inherits from `IBMQDevice` because its [connection](https://github.com/PennyLaneAI/pennylane-qiskit/blob/2344aa4a866e7f8614fc87df3004f645156838ee/pennylane_qiskit/ibmq.py#L77)) which I don't think applies to this test, so we can probably remove it from this step and the one below 👍

Reply:
Actually I think we should add an IBMQ device to the test instead, since I'm realizing different devices have different import errors that might be raised before the more informative error message we want to come up